examples/vertx-opentracing/genorders.sh

```diff
@@ -17,24 +17,24 @@
 
 while :
 do
-	declare -a AccountIds=('joe' 'sarah' 'fred' 'jane' 'brian' 'steve');
```

Comment:
Why are you changing indentation to 2 spaces instead of 4 which is used in other files?